public/regdesk.html

```diff
@@ -240,9 +240,9 @@ <h5 class="modal-title">Printer Settings For: <span id="printerSettingsSerial"><
                 </div>
               </div>
               <div class="col-md-4">
-                <label for="printerSettingsDarkness" class="col-sm-6 col-form-label">Darkness</label>
+                <label for="printerSettingsDarkness" class="form-label">Print Darkness</label>
                 <select id="printerSettingsDarkness" class="form-select" aria-label="Darkness">
-                  <option>1</option>
+                  <option value="1">1 - Lightest</option>
                   <option>2</option>
```

Comment:
Only just realizing, should these have a default value too?

Reply:
The value should always be pulled from the printer configuration and that internally sets a default, I don't think we need one here.